sdk/tests/deqp/framework/opengl/simplereference/sglrReferenceContext.js

```diff
@@ -4442,12 +4442,6 @@ goog.scope(function() {
                  target == gl.TEXTURE_CUBE_MAP_POSITIVE_Y ||
                  target == gl.TEXTURE_CUBE_MAP_NEGATIVE_Z ||
                  target == gl.TEXTURE_CUBE_MAP_POSITIVE_Z) {
-            // Validate size and level.
-            if (this.conditionalSetError(width != height || width > this.m_limits.maxTextureCubeSize || depth != 1, gl.INVALID_VALUE))
-                return;
-            if (this.conditionalSetError(level > Math.floor(Math.log2(this.m_limits.maxTextureCubeSize)), gl.INVALID_VALUE))
-                return;
-
```

Comment:
This is just wrong.  Checking width == height for texsubimage calls causes all cubemap reference image generation to fail.  The C++ code doesn't have these checks - likely copied over from teximage part and forgot to delete.

https://code.google.com/p/chromium/codesearch#chromium/src/third_party/deqp/src/framework/opengl/simplereference/sglrReferenceContext.cpp&l=1344